website/content/en/where.adoc

```diff
@@ -156,9 +156,7 @@ Uppercase has special meaning. For example:
 * a first _beta release_ is not a (production) `RELEASE`.
 The word `CURRENT` is sometimes a source of confusion:
```

Comment:
I wonder if this sentence should be spaced before, so it is not inside the bullet? Looks a little strange.

Reply:
Yes, seems we need an empty line there